lib/scrabble.rb

```diff
@@ -1,18 +1,47 @@
+require './lib/player'
 class Scrabble
+  attr_reader :players
 
-  def score(word)
-    1
+
+  def initialize(player = Player.new("Bob"))
+    @players = [player]
+    @point_values ={
+              "A"=>1, "B"=>3, "C"=>3, "D"=>2,
+              "E"=>1, "F"=>4, "G"=>2, "H"=>4,
+              "I"=>1, "J"=>8, "K"=>5, "L"=>1,
+              "M"=>3, "N"=>1, "O"=>1, "P"=>3,
+              "Q"=>10, "R"=>1, "S"=>1, "T"=>1,
+              "U"=>1, "V"=>4, "W"=>4, "X"=>8,
+              "Y"=>4, "Z"=>10
+            }
   end
 
-  def point_values
-    {
-      "A"=>1, "B"=>3, "C"=>3, "D"=>2,
-      "E"=>1, "F"=>4, "G"=>2, "H"=>4,
-      "I"=>1, "J"=>8, "K"=>5, "L"=>1,
-      "M"=>3, "N"=>1, "O"=>1, "P"=>3,
-      "Q"=>10, "R"=>1, "S"=>1, "T"=>1,
-      "U"=>1, "V"=>4, "W"=>4, "X"=>8,
-      "Y"=>4, "Z"=>10
-    }
+  def player_turn(player, word)
+    player.score += word.score
   end
+
+
+  def score(word)
+    letters = word.chars
+    total_score = letters.map do |letter|
+      @point_values[letter.upcase].to_i
+    end
+    # binding.pry
+    sum = 0
+    total_score.each do |score|
+      sum += score
+      # binding.pry
+    end
+    sum
+  end
```

Comment:
This looks good. Check on `#sum` in Ruby 2.4, or `#reduce(:+)` in earlier versions of Ruby.